EnvironmentSetup/Azure/Source/strimzi-kafka-deployment/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -118,7 +116,7 @@ spec:
           resources:
             limits:
               cpu: 1000m
-              memory: 384Mi
+              memory: 2Gi
```

Comment:
As per strimzi source code / recommendation this value is `memory: 384Mi` - why has it been changed to `2Gi` ? how was this increased value chosen?

Reply:
For some reason kafka-persistent-zookeeper pods are exited with OOMKilled when initializing and it caused failure of the script. I increased the limits to 1Gi, the result was same. So I increased it to 2Gi.

Yes, in default they're 384Mi in the helm chart. 
https://github.com/strimzi/strimzi-kafka-operator/blob/0.29.0/helm-charts/helm3/strimzi-kafka-operator/values.yaml#L139-L142